keanu-project/src/main/java/io/improbable/keanu/vertices/bool/BoolVertex.java

```diff
@@ -30,6 +30,18 @@ public BoolVertex(long[] initialShape) {
         super(initialShape);
     }
 
+    @Override
+    public BoolVertex print() {
```

Comment:
This has the same functionality as the print on the `Vertex` class so I think you don't need to override. See my comment on `Vertex.java` for something that should work.